include/jitk/statistics.hpp

```diff
@@ -71,12 +82,13 @@ class Statistics {
     std::chrono::duration<double> time_copy2dev{0};
     std::chrono::duration<double> time_copy2host{0};
     std::chrono::duration<double> time_ext_method{0};
+    std::map<std::string, KernelStats> time_per_kernel;
 
     std::chrono::duration<double> wallclock{0};
     std::chrono::time_point<std::chrono::steady_clock> time_started{std::chrono::steady_clock::now()};
 
-    Statistics(bool enabled) : enabled(enabled), print_on_exit(enabled) {}
-    Statistics(bool enabled, bool print_on_exit) : enabled(enabled), print_on_exit(print_on_exit) {}
+    Statistics(bool enabled, bool verbose) : enabled(enabled), print_on_exit(enabled), verbose(verbose) {}
```

Comment:
I think that `Statistics` should take `const ConfigParser &config` instead of `bool enabled, bool verbose`.

Reply:
Why is there even a separate flag for `enabled` and `print_on_exit`? As far as I can see, they are always the same value.